cip-0001.md

```diff
@@ -45,7 +45,7 @@ Standards Track CIPs consist of two parts, a design document and a reference imp
 
 For a CIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-To be accepted, the CIP's pull request must be approved ("ACKed") by at least two project maintainers. A project maintainer is defined as an individual that has commit access to the (counterparty-lib)[https://github.com/CounterpartyXCP/counterparty-lib] repository. If there is only one maintainer, obviously only his or her ACK is necessary.
+To be accepted, the CIP's pull request must be approved ("ACKed") by a majority of project maintainers. A project maintainer is defined as an individual that has commit access to the (counterparty-lib)[https://github.com/CounterpartyXCP/counterparty-lib] repository, and has made at least one commit in the last six month period. If there is only one maintainer, obviously only his or her ACK is necessary.
```

Comment:
This seems reasonable to me.  We should make the sunset length 1 year to be safe.